src/tests/test_magic.py

```diff
@@ -1305,3 +1305,194 @@ def test_interact_and_missing_ipywidgets_installed(ip):
             "%sql --interact my_variable SELECT * FROM author LIMIT {{my_variable}}"
         )
         assert isinstance(out.error_in_exec, ModuleNotFoundError)
+
+
+@pytest.mark.parametrize(
+    "file_content, expect, revert",
+    [
+        (
+            """
+[tool.jupysql.SqlMagic]
+autocommit = false
+autolimit = 1
+style = "RANDOM"
+""",
+            """Found pyproject.toml from '%s'
+Settings changed:
+
++------------+--------+
+|   Config   | value  |
++------------+--------+
+| autocommit | False  |
```

Comment:
can we check columns only instead of harcoding the entire structure?

Reply:
Extracted rows instead and check them in the assert statement below using regular expression.